src/components/headerdiv.jsx

```diff
@@ -0,0 +1,51 @@
+const HeaderDiv = () => {
+  return (
+    <div class="header flex h-[10vh] w-full items-center justify-between">
+      <div class="header-texts flex h-[80%] w-4/5 items-center justify-between">
+        <h1 class="calc-text text-3xl">calc</h1>
+        <p>THEME</p>
+      </div>
```

Comment:
I agree that this can be an header component, however, the below should be a `ThemeSwitcher` component (or similar name)